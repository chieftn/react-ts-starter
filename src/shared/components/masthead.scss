.masthead {
    height: 45px;
    background-color: #3c3c41;
    justify-content: space-between;
    display: flex;
    width: 100vw;
    .application-name {
        margin-top: 10px;
        margin-left: 5px;
        font-size: 18px;
        color: white;
        font-weight: 600;
    }
    .application-account {
        color: white;
    }
}
