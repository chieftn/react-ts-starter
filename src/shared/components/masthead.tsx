import * as React from 'react';
import { useTranslation } from 'react-i18next';
import { ResourceKeys } from '../../localization/resourceKeys';
import { Account } from '../../apps/account/components/account';
import './masthead.scss'

export const Masthead: React.FC = () => {
    const { t } = useTranslation();

    return (
        <div className="masthead">
            <div className="application-name">{t(ResourceKeys.common.productName)}</div>
            <span style={{ marginLeft: 10, backgroundColor: 'white'}}>
                <Account/>
            </span>
        </div>
    );
};
