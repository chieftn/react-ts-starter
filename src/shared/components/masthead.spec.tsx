import * as React from 'react';
// import { render } from '@testing-library/react';
import { create } from 'react-test-renderer';
import { Masthead } from './masthead';

describe('masthead', () => {
    it('matches snapshot', () => {
        const component = create(<Masthead/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
