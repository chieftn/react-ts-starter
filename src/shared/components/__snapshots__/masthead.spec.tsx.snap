// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`masthead matches snapshot 1`] = `
<div
  className="masthead"
>
  <div
    className="application-name"
  >
    "common.productName"
  </div>
</div>
`;
